src/v2/repositories/implementations/XcmRepository.ts

```diff
@@ -215,6 +240,17 @@ export class XcmRepository implements IXcmRepository {
       ],
     };
 
+    console.log('XcmRepository');
```

Comment:
Is this still needed or it was just debug log?

Reply:
Thanks, I've fixed in [src/v2/repositories/implementations/XcmRepository.ts](https://github.com/AstarNetwork/astar-apps/pull/758/files/ecd8fd448d629222924992502e8b06c6817d23da#diff-4cba2fed53f9b329ee1c759339d4d0d7e7d3a56f39081ee13513286accab7f2b) commit.